lib/atomic-management.js

```diff
@@ -270,20 +309,17 @@ export default {
     const projectPaths = atom.project.getPaths();
     this.projectConfigs = [];
     projectPaths.forEach(projectPath => {
-      var config = glob.sync(`${projectPath}/atomconfig.@(cson|json)`);
+      var config = glob.sync(`${projectPath}/${FILE_NAME}.@(cson|json)`);
```

Comment:
This way when we change from `.atomconfig` to `./atom/atomconfig` it'll be a little easier